.github/workflows/release.yml

```diff
@@ -141,3 +141,28 @@ jobs:
             $RC \
             $LATEST \
             release-artifacts/*
+
+  publish-crate:
```

Comment:
Does this need to depend on release?

I guess technically there are separate and we pushed the tag at this point already so I guess having a crate release without a github release is fine.

Reply:
It is fine like that.